golang/defaults.yaml

```diff
@@ -12,15 +12,15 @@ golang:
     name: golang
     use_upstream_repo: false
     use_upstream_archive: true
+    uri: https://storage.googleapis.com/golang
     archive:
       name: /usr/local
-      uri: https://storage.googleapis.com/golang
       source: null
       source_hash: null
       trim_output: true
-      archive_suffix: tar.gz
       archive_format: tar
       enforce_toplevel: true
+    archive_suffix: tar.gz
```

Comment:
This was a fix for "@myii  This was the fix for "TypeError: unzip() got an unexpected keyword argument 'archive_suffix'" too, which I also saw on Windows 10.  I thought these were fixed?